packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -37,8 +39,10 @@ export default (siteVars): ChatMessageVariables => ({
   offset: pxToRem(100),
   padding: pxToRem(16),
   authorMarginRight: pxToRem(12),
-  authorFontWeight: siteVars.fontWeightBold,
+  authorColor: siteVars.gray02, // will be gray[500] with new palette
+  authorFontWeight: siteVars.fontWeightRegular,
   headerMarginBottom: pxToRem(2),
+  contentColor: '#252423', // will be gray[750] with new palette
```

Comment:
There is a `color` variable here too. I wonder if we don't need this variable if the color prop above is set to this.

Reply:
But `color` is `rgb(64, 64, 64)` that points to `#404040`...